types/ember/index.d.ts

```diff
@@ -1837,7 +1839,7 @@ declare module 'ember' {
              * A hook you can implement to convert the URL into the model for
              * this route.
              */
-            model<T>(params: {}, transition: Transition): T | Rsvp.Promise<T>;
+            model(params: {}, transition: Transition): any;
```

Comment:
why is this necessary?

Reply:
@bttf The type parameter makes no sense here, and `any | Rsvp.Promise<any>` is equivalent to just `any`. See typed-ember/ember-typings#61